scootapi/setup/builder.go

```diff
@@ -0,0 +1,69 @@
+package setup
```

Comment:
Yup; I agree. I thought about making there be an interface for just the command to run, but then felt like that was too much abstraction... WDYT? Is it worth unifying these?

(I'd tend to say "let's unify them if/when we see them drift apart")